sql/catalyst/src/main/scala/org/apache/spark/sql/types/HiveStringType.scala

```diff
@@ -47,9 +47,7 @@ object HiveStringType {
     case MapType(kt, vt, nullable) =>
       MapType(replaceCharType(kt), replaceCharType(vt), nullable)
     case StructType(fields) =>
-      StructType(fields.map { field =>
-        field.copy(dataType = replaceCharType(field.dataType))
-      })
+      StructType(fields.map(f => f.copy(dataType = replaceCharType(f.dataType))))
```

Comment:
Ur, is this relevant to this `HiveVoidType` PR?

Reply:
@LantaoJin . I understand that you want to match the style, but a PR should be minimal and should not touch irrelevant parts.

Reply:
Ok, get the rule now. I will rollback this line